testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -734,10 +733,29 @@ def __init__(self):
 
     def test_atomname_alignment(self):
         # Our PDBWriter adds some stuff up top, so line 1 happens at [4]
-        assert_(self.writtenstuff[4].startswith("ATOM      1  H5T GUA"))
-        assert_(self.writtenstuff[8].startswith("ATOM      5 H5'' GUA"))
+        refs = ("ATOM      1  H5T",
+                "ATOM      2  CA ",
+                "ATOM      3 CA  ",
+                "ATOM      4 H5''",)
+        for written, reference in zip(self.writtenstuff[4:], refs):
+            assert_equal(written[:16], reference)
 
     def test_atomtype_alignment(self):
         result_line = ("ATOM      1  H5T GUA R   1       7.974   6.430   9.561"
                        "  1.00  0.00      RNAA H\n")
         assert_equal(self.writtenstuff[4], result_line)
+
+def test_deduce_PDB_atom_name():
+    Pair = mda.coordinates.PDB.Pair
+    def _test_PDB_atom_name(atom, ref_atom_name):
+        name = mda.coordinates.PDB.PrimitivePDBWriter._deduce_PDB_atom_name(atom)
+        assert_equal(name, ref_atom_name)
+    test_cases = ((Pair('ASP', 'CA'), ' CA '),  # Regular protein carbon alpha
```

Comment:
Are you using `Pair` in the test to have a minimal "mock" of the Atom-class API?  Neat trick. But you should comment that to avoid confusion about it in the future.